src/Parsers/ASTSystemQuery.cpp

```diff
@@ -174,6 +174,7 @@ void ASTSystemQuery::formatImpl(const FormatSettings & settings, FormatState & s
         case Type::STOP_CLEANUP:
         case Type::START_CLEANUP:
         case Type::UNLOAD_PRIMARY_KEY:
+        case Type::LOAD_PRIMARY_KEY:
```

Comment:
```cpp
        case Type::LOAD_PRIMARY_KEY:
        case Type::UNLOAD_PRIMARY_KEY:
```

Reply:
Fixed

Reply:
I still see the `LOAD_PRIMARY_KEY` after the `UNLOAD_PRIMARY_KEY`